luda-editor/luda-editor-client/src/app/sequence_list/rounded_rectangle.rs

```diff
@@ -0,0 +1,50 @@
+use std::convert;
+
+use super::{SequenceList, RECT_RADIUS};
+use namui::{Color, RectFill, RectParam, RectRound, RectStyle, RenderingTree, Wh};
+
+pub enum RoundedRectangleColor {
+    DarkGray,
+    Gray,
+    LightGray,
+    Blue,
+    // Red,
+    // White,
+}
+
+impl SequenceList {
+    pub fn render_rounded_rectangle(
+        &self,
+        wh: Wh<f32>,
+        color: RoundedRectangleColor,
+    ) -> RenderingTree {
+        namui::rect(RectParam {
+            x: 0.0,
+            y: 0.0,
+            width: wh.width,
+            height: wh.height,
+            style: RectStyle {
+                stroke: None,
+                fill: Some(RectFill {
+                    color: color.into(),
+                }),
+                round: Some(RectRound {
+                    radius: RECT_RADIUS,
+                }),
+            },
+        })
+    }
+}
+
+impl convert::Into<Color> for RoundedRectangleColor {
+    fn into(self) -> Color {
+        match self {
+            RoundedRectangleColor::DarkGray => Color::grayscale_f01(0.3),
+            RoundedRectangleColor::Gray => Color::grayscale_f01(0.5),
+            RoundedRectangleColor::LightGray => Color::grayscale_f01(0.8),
+            RoundedRectangleColor::Blue => Color::from_u8(107, 185, 240, 255),
+            // RoundedRectangleColor::Red => Color::from_u8(242, 38, 19, 255),
+            // RoundedRectangleColor::White => Color::WHITE,
```

Comment:
I think you can remove this comment or uncomment.

Reply:
It will be used to make features in issue.
But if I don't comment it. It shows me `dead_code` warning. It was very annoying.
So I commented it and I'll uncomment it when I use it